src/components/Editor/Runners/HtmlRunner/HtmlRunner.js

```diff
@@ -102,24 +134,38 @@ function HtmlRunner() {
       const projectFile = projectCode.filter(
         (file) => `${file.name}.${file.extension}` === hrefNode.attrs.href
       );
+      // remove target blanks
+      if (hrefNode.attrs?.target === "_blank") {
+        hrefNode.removeAttribute("target");
+      }
+
+      let onClick;
+
       if (!!projectFile.length) {
-        const projectFileBlob = getBlobURL(
-          projectFile[0].content,
-          `text/${projectFile[0].extension}`
-        );
-        hrefNode.setAttribute("href", projectFileBlob);
+        if (parentTag(hrefNode, "head")) {
+          const projectFileBlob = getBlobURL(
+            cssProjectImgs(projectFile[0]).content,
+            `text/${projectFile[0].extension}`
+          );
+          hrefNode.setAttribute("href", projectFileBlob);
+        } else {
+          // eslint-disable-next-line no-script-url
+          hrefNode.setAttribute("href", "javascript:void(0)");
```

Comment:
📓 annoying but necessary to all us to treat links as links without any impact to the user, setting to `#` will ping them to the top of the page and setting to `undefined` strips link styling. Used same approach for reloads (i.e. links to another file) as with external links